sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java

```diff
@@ -501,7 +502,9 @@ public Registration(Class<T> proxyClass, List<PropertyDescriptor> beanInfo) {
   private static final DefaultDeserializationContext DESERIALIZATION_CONTEXT =
       new DefaultDeserializationContext.Impl(MAPPER.getDeserializationContext().getFactory())
           .createInstance(
-              MAPPER.getDeserializationConfig(), new TokenBuffer(MAPPER, false).asParser(), null);
+              MAPPER.getDeserializationConfig(),
+              new TokenBuffer(MAPPER, false).asParser(),
+              new InjectableValues.Std());
```

Comment:
Could you help me understand what does adding `InjectableValues.Std()` change?

Reply:
nothing, but since we got bit by parameters being null in the past, I figured I might as well pass in something for this one as well.

Reply:
It will just lead to a different exception being thrown [no injectable values](https://github.com/joansmith/jackson-databind/blob/230a19d527df7dc0c05017429bcb7aa1cbbe41ef/src/main/java/com/fasterxml/jackson/databind/DeserializationContext.java#L375) vs [can't find the value](https://github.com/joansmith/jackson-databind/blob/230a19d527df7dc0c05017429bcb7aa1cbbe41ef/src/main/java/com/fasterxml/jackson/databind/InjectableValues.java#L75) since the InjectableValues is empty.